geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/index/HBaseFeatureIndex.scala

```diff
@@ -41,10 +44,16 @@ object HBaseFeatureIndex extends HBaseIndexManagerType {
 
   val DataColumnQualifier = Bytes.toBytes("d")
   val DataColumnQualifierDescriptor = new HColumnDescriptor(DataColumnQualifier)
+
+  case class ScanConfig(hbaseFilters: Seq[HBaseFilter],
+                        columnFamily: Array[Byte],
```

Comment:
column family is already incorporated in the `ranges`, so I think we can remove it here as it's not used